File: basic/Override/1.py
class C1 :
    def m(self) :
        return 'parent'

class C2(C1) :
    def m(self) :
    # python에서 super는 부모의 클래를 가리킨다
        return super().m() + ' child'
    # 비어있는 클래스 표시
    pass

o = C2()
print(o.m())

File: basic/Override/1.rb
class C1
  def m()
    return 'parent'
  end
end

class C2 < C1
  def m()
    # ruby에서는 super는 메소드를 가리킨다
    return super() + ' child'
  end
end

o = C2.new()
p o.m()
